serverless/tensorflow/matterport/mask_rcnn/nuclio/function.yaml

```diff
@@ -97,26 +97,22 @@ spec:
   runtime: 'python:3.6'
   handler: main:handler
   eventTimeout: 30s
-  env:
-    - name: MASK_RCNN_DIR
-      value: /opt/nuclio/Mask_RCNN
+
   build:
     image: cvat/tf.matterport.mask_rcnn
-    baseImage: tensorflow/tensorflow:2.1.0-py3
+    baseImage: tensorflow/tensorflow:1.13.1-py3
```

Comment:
Before you were installing `RUN pip install tensorflow==1.13.1` . RUN can't be overwritten using nuctl for gpu. If I use tensorflow 2.1-GPU I was getting some errors. Besides the mask-rcnn code is in tensorflow 1.x. It is better to have the docker for 1.x .
There was no reason not to go with v 1.15.5, we might be able to switch to that version probably. I just needed a version that works fine. I used 1.13.1 because it was being installed on line 119.